app/components/Table/index.js

```diff
@@ -273,7 +273,7 @@ export default class Table extends Component<Props, State> {
 
     const match = Object.keys(this.state.filters).filter((key) => {
       const { inlineFiltering = true, filterMapping = (val) => val } =
-        this.props.columns.find((col) => col.dataIndex == key) || {};
+        this.props.columns.find((col) => col.dataIndex === key) || {};
```

Comment:
Sure this logic stays 100% similar and works if the dataIndex is a number?

Reply:
As far as i can tell, dataIndex is always a string (In our codebase). If the dataIndex is a "number", it will probably just be typeof string anyways. :shrug: